src/components/EmptyStateComponent/index.tsx

```diff
@@ -24,10 +25,12 @@ function EmptyStateComponent({
     subtitle,
     headerStyles,
     headerContentStyles,
+    lottieWebViewStyles,
     minModalHeight = 400,
 }: EmptyStateComponentProps) {
     const styles = useThemeStyles();
     const [videoAspectRatio, setVideoAspectRatio] = useState(VIDEO_ASPECT_RATIO);
+    const {isSmallScreenWidth} = useResponsiveLayout();
```

Comment:
NAB because it seems like all the styles are right in the screen recordings, but I think we might be able to use `shouldUseNarrowLayout` here, and generally that should be the default go-to instead of `isSmallScreenWidth`